setup.py

```diff
@@ -64,7 +64,8 @@ def get_dependency():
         'uncertainty-calibration==0.0.8',
         'dill==0.3.3',
         'scikit-image==0.17.2',
-        'prettytable==2.2.1'
+        'prettytable==2.2.1',
+        'nltk'
```

Comment:
include version number for new dependencies. Leaving them off causes problems since we have so many that all need to work together